sql/test_relation_size.sql

```diff
@@ -0,0 +1,30 @@
+create temp table t1(i int);
+insert into t1 select generate_series(1, 10000);
+select diskquota.relation_size('t1', true);
+select pg_table_size('t1');
+
+create table t2(i int);
+insert into t2 select generate_series(1, 10000);
+select diskquota.relation_size('t2', false);
+select pg_table_size('t2');
+
+-- start_ignore
+\! mkdir /tmp/test_spc
+-- end_ignore
+DROP TABLESPACE  IF EXISTS test_spc;
+CREATE TABLESPACE test_spc LOCATION '/tmp/test_spc';
```

Comment:
Nit: keep coding style consistent.
```suggestion
drop tablespace  if exists test_spc;
create tablespace test_spc location '/tmp/test_spc';
```